client/lib/plans/features-list.tsx

```diff
@@ -1769,6 +1780,22 @@ export const FEATURES_LIST: FeatureList = {
 		getAlternativeTitle: () => '0%',
 		getFeatureGroup: () => FEATURE_GROUP_PAYMENT_TRANSACTION_FEES,
 	},
+	[ FEATURE_50GB_STORAGE_ADD_ON ]: {
+		getSlug: () => PRODUCT_1GB_SPACE,
```

Comment:
Is this correct btw? `...1GB...`?

Reply:
Yes it is. It's described further in the PR description

```
The additional storage is a quantity-based product, covered in this PT: pau2Xa-4MT-p2

It's implemented as an x * 1 GB (where x is quantity)
```